apps/passport-server/src/database/queries/pretix_config/insertConfiguration.ts

```diff
@@ -44,7 +45,9 @@ export async function insertPretixEventConfig(
   const result = await sqlQuery(
     db,
     `insert into pretix_events_config(pretix_organizers_config_id, active_item_ids, event_id, superuser_item_ids) ` +
-      `values ($1, $2, $3, $4) returning id`,
+      `values ($1, $2, $3, $4) ` +
+      `on conflict (event_id) do update set pretix_organizers_config_id = $1, active_item_ids = $2, superuser_item_ids = $4 ` +
+      `returning id`,
```

Comment:
does this need to be tested? cc @robknight

Reply:
This function is only ever called from tests (`devconnectdb.spec.ts` and `devconnect.spec.ts`).

For local use, we've been using a set of SQL queries that insert the config into the DB. I'm not actually sure how the production or staging databases were populated - presumably manually?

Perhaps it's worth adding a test to `devconnectdb.spec.ts` to verify that inserting conflicting config works as expected, but we don't currently have any application code that will call it.

Reply:
Note: This code is staying as is, because if a dev runs the `new-dev-event` command from the CLI multiple times, we don't want to keep making new rows in the table